doc/admin-guide/files/records.yaml.en.rst

```diff
@@ -4667,6 +4667,23 @@ removed in the future without prior notice.
    that |TS| is willing to store. The value MUST be at least 2.
    Transport Parameter.
 
+.. ts:cv:: CONFIG proxy.config.quic.max_recv_udp_payload_size_in INT 65527
+
+   This value will be advertised as ``max_udp_payload_size`` Transport Parameter.
+
+.. ts:cv:: CONFIG proxy.config.quic.max_recv_udp_payload_size_out INT 65527
+
+   This value will be advertised as ``max_udp_payload_size`` Transport Parameter.
```

Comment:
Two configs with exactly the same description.

Reply:
Yes, they reference to the same transport parameter.  I think this eventually will contain more details, or even a link to the current rfc definition.

Reply:
I suggest adding "for incoming/outgoing" packets.